client/client-lib/src/api.rs

```diff
@@ -107,121 +106,72 @@ impl ApiError {
     /// Returns `true` if queried outpoint isn't ready yet but may become ready later
     pub fn is_retryable(&self) -> bool {
         match self {
-            ApiError::HttpError(e) => e.status() == Some(StatusCode::NOT_FOUND),
+            // TODO
+            // ApiError::HttpError(e) => e.status() == Some(StatusCode::NOT_FOUND),
```

Comment:
Did you want to resolve this before merge?

Reply:
yes, otherwise polling won't work

Reply:
Second PR would be using notifications/subscriptions to get rid of that I assume?